app/models/publication.rb

```diff
@@ -76,10 +76,13 @@ def add_author_emails(people)
     end
   end
 
+  # :recruiting_authors state for when abstract and author email are fetched from WoS page
+  # :bypass_fetch_recruiting_authors state for when abstract and author email are loaded from export file
   aasm do
     state :initialized, initial: true
     state :fetching_authors
     state :recruiting_authors
+    state :bypass_fetch_recruiting_authors
```

Comment:
All the code for this state is going to cause problems in the future. Whenever any publication attempts to recruit authors, it won't fail if they don't have an email and the workflow will continue forward. This won't be very good for publications that actually have a problem finding the author's email and won't just affect these CSV ingested ones.